src/Measurements.jl

```diff
@@ -130,6 +143,9 @@ end
 
 @static if !isdefined(Base, :get_extension)
     function __init__()
+        @static if Base.VERSION >= v"1.6" # minimum version supported by Symbolics 5.x.x
+          @require Symbolics="0c5d862f-8b57-4792-8d23-62f2024744c7" include("../ext/MeasurementsSymbolicsExt.jl")
+        end
```

Comment:
Let's keep this v1.9-only and don't add even more Requires hooks: https://github.com/JuliaPhysics/Measurements.jl/pull/145#discussion_r1199801162
```suggestion
```